src/gui/advancedsettings.cpp

```diff
@@ -349,42 +353,75 @@ void AdvancedSettings::loadAdvancedSettings()
     spinBoxCache.setValue(session->diskCacheSize());
     updateCacheSpinSuffix(spinBoxCache.value());
     addRow(DISK_CACHE, tr("Disk cache"), &spinBoxCache);
+    item(DISK_CACHE, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("The disk write and read cache. If this is set to -1, the cache size is automatically "
+                    "set to the amount of physical RAM available in the machine divided by 8. "
+                    "If the amount of physical RAM cannot be determined, it's set to 16 MiB")));
```

Comment:
@thalieht:  Note also that (in qBittorrent) if this is **set to 0**, the cache is disabled entirely.

Reply:
Fixed.